Support usingServer options for selenium-webdriver

Ineigo opened the pull request:
Use:
```JSON
"testEnvironmentOptions": {
      "browser": "chrome",
      "seleniumAddress": "http://127.0.0.1:4444/wd/hub"
}
```

Ineigo committed:
Support usingServer options for selenium-webdriver

evantahler commented:
You can se the server via ENVIRONMENT variables, ie: `SELENIUM_REMOTE_URL="http://localhost:4444/wd/hub" jest`

alexeyraspopov commented on packages/jest-environment-webdriver/modules/WebDriverEnvironment.js:
```diff
@@ -6,14 +6,17 @@ class WebDriverEnvironment extends NodeEnvironment {
-      .forBrowser(this.browserName)
-      .build();
+    
+    let driver = await new Builder();
```
You can avoid `await` here, since there is no `build()` in the chain yet.

Ineigo replied:
Thank you!

alexeyraspopov commented on packages/jest-environment-webdriver/modules/WebDriverEnvironment.js:
```diff
@@ -6,14 +6,17 @@ class WebDriverEnvironment extends NodeEnvironment {
     super(config);
     const options = config.testEnvironmentOptions || {};
     this.browserName = options.browser || 'chrome';
+    this.usingServer = options.usingServer || null;
```
I'd call the option `seleniumAddress` so it would have some context when written in Jest config. What do you think?

Ineigo replied:
I like it )

alexeyraspopov reviewed (commented):
@Ineigo, thank you for your contribution! This is definitely something I considered doing but didn't have enough time. I'm glad you were able to made such improvement.

I left a couple of minor comments that can help us to finalize the solution. Meanwhile, @evantahler's suggestion to use environment variable works due to the way how WebDriver gets configured.

I apologize for such late response. I need to figure what happened to my notifications and why did I miss your pull request.

Ineigo committed:
Rename usingServer -> seleniumAddress & fix await

New use:
```JSON
"testEnvironmentOptions": {
      "browser": "chrome",
      "seleniumAddress": "http://127.0.0.1:4444/wd/hub"
}
```

alexeyraspopov reviewed (approved):
This is great, thank you! I'm going to update README and publish a new version later today.

alexeyraspopov commented:
Published [`jest-environment-webdriver@0.2.0`](https://github.com/alexeyraspopov/jest-webdriver/releases/tag/jest-environment-webdriver%400.2.0).
